Geometry/CMSCommonData/python/cmsExtendedGeometry2017Plan1XML_cfi.py

```diff
@@ -230,7 +230,7 @@
         'Geometry/MuonCommonData/data/Run2/mb3.xml',
         'Geometry/MuonCommonData/data/Run2/mb4.xml',
         'Geometry/MuonCommonData/data/design/muonYoke.xml',
-        'Geometry/MuonCommonData/data/v2/mf.xml',
+        'Geometry/MuonCommonData/data/v6/mf.xml',
```

Comment:
@bsunanda - this removes GE11 extrusion, but introduces an overlap of ME1RingP with SupportTubeForHE:
```
 = Overlap ov02148: mf:MEP/mf:ME1RingP_1 overlapping mf:MEP/mfshield:SupportTubeForHE_1 ovlp=3.3
 - first volume: mf:ME1RingP at position:
matrix  - tr=0  rot=0  refl=0  scl=0
  1.000000    0.000000    0.000000    Tx =   0.000000
  0.000000    1.000000    0.000000    Ty =   0.000000
  0.000000    0.000000    1.000000    Tz =   0.000000
*** Shape mf:ME1Ring: TGeoPcon ***
    Nz    = 4
    phi1  =     0.00000
    dphi  =   360.00000
     plane 0: z=  564.10000 Rmin=  115.00000 Rmax=  280.00000
     plane 1: z=  567.40000 Rmin=  115.00000 Rmax=  280.00000
     plane 2: z=  567.40000 Rmin=   85.00000 Rmax=  280.00000
     plane 3: z=  634.50000 Rmin=   85.00000 Rmax=  280.00000
 Bounding box:
*** Shape mf:ME1Ring: TGeoBBox ***
    dX =   280.00000
    dY =   280.00000
    dZ =    35.20000
    origin: x=    0.00000 y=    0.00000 z=  599.30000
 - second volume: mfshield:SupportTubeForHE at position:
matrix  - tr=0  rot=0  refl=0  scl=0
  1.000000    0.000000    0.000000    Tx =   0.000000
  0.000000    1.000000    0.000000    Ty =   0.000000
  0.000000    0.000000    1.000000    Tz =   0.000000
*** Shape mfshield:SupportTubeForHE: TGeoPcon ***
    Nz    = 7
    phi1  =     0.00000
    dphi  =   360.00000
     plane 0: z=  554.10000 Rmin=   67.50000 Rmax=  251.00000
     plane 1: z=  564.10000 Rmin=   67.50000 Rmax=  251.00000
     plane 2: z=  564.10000 Rmin=   67.50000 Rmax=  130.00000
     plane 3: z=  567.40000 Rmin=   67.50000 Rmax=  130.00000
     plane 4: z=  567.40000 Rmin=   67.50000 Rmax=   85.00000
     plane 5: z=  659.00000 Rmin=   67.50000 Rmax=   85.00000
     plane 6: z=  683.50000 Rmin=   70.10000 Rmax=   85.00000
 Bounding box:
*** Shape mfshield:SupportTubeForHE: TGeoBBox ***
    dX =   251.00000
    dY =   251.00000
    dZ =    64.70000
    origin: x=    0.00000 y=    0.00000 z=  618.80000


```